app/mailers/user_mailer.rb

```diff
@@ -0,0 +1,11 @@
+class UserMailer < ActionMailer::Base
+  default from: "no-reply@bpac.com"
+  layout 'mailer'
+
+  def invite(user_id)
```

Comment:
Can we rename this to something like `invitation_email`? `invite` makes it sound like the method will send the invitation email, but it really doesn't send anything.